pages/index.jsx

```diff
@@ -3,10 +3,13 @@ import { NewRequest } from 'components/NewRequest';
 import React, { useEffect, useState } from 'react';
 import { Row, Col } from 'antd';
 import VideoCard from 'components/VideoCards/VideoCard';
+import { getAgentURL } from 'common-util/Contracts';
 
 export const getServerSideProps = async () => {
   try {
-    const response = await fetch(`${process.env.NEXT_PUBLIC_AGENT_URL}/responses`, { mode: 'no-cors' });
+    /// we server side render for 100
+    const agentURL = getAgentURL(100);
```

Comment:
This is the problem! We need the actual chainID, not the default one... but here we are server side...